modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -38,10 +38,27 @@ If you disable the Inbound Traffic tab, changes can take several minutes to comp
 . In the *Resource Allocation* section:
 +
 The inbound load balancer runs on each controller node configured in Runtime Fabric. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
+
+.. Select a mode.
+
+*** *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+
+*** *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If you are setting up a Runtime Fabric with dedicated nodes, include only the dedicated node IP addresses. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated. 
```

Comment:
Points we should consider addressing:
 - what node is available? (internal load balancer node)
 - unsure on the "dedicated node IP addresses" -- is that something the user has to put in (I don't think that's true, but want to confirm)
 - what is the intent of the last sentence? (unsure)

Reply:
Regarding the 2nd bullet -- this may be in reference to configuring an external load balancer.

Reply:
Reworded this but the info pertaining to your second bullet wasn't updated yet.